src/main/java/ch/njol/skript/expressions/ExprLootTableSeed.java

```diff
@@ -0,0 +1,90 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.event.Event;
+import org.bukkit.loot.Lootable;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Loot Table Seed")
+@Description("Returns the seed of a loot table.")
+@Examples({
+	"seed of loot table of block",
+	"set seed of loot table of block to 123456789"
+})
+@Since("INSERT VERSION")
+public class ExprLootTableSeed extends SimplePropertyExpression<Object, Long> {
+
+	static {
+		register(ExprLootTableSeed.class, Long.class, "loot[ ]table seed[s]", "entities/blocks");
+	}
```

Comment:
The examples don't look like what you're registering.

As well as that, I think table can be optional for the syntax, just `loot seed of X` sounds okay.

Reply:
oopsies